transport/errors/errormapper.go

```diff
@@ -8,24 +8,24 @@ import (
 
 func ToError(err error) *Error {
 	switch {
-	case isUnauthorizedError(err):
+	case IsUnauthorizedError(err):
 		return NewUnauthorizedError().Wrap(err)
-	case isForbiddenError(err):
+	case IsForbiddenError(err):
 		return NewForbiddenError().Wrap(err)
-	case isNotFoundError(err):
+	case IsNotFoundError(err):
 		return NewNotFoundError().Wrap(err)
-	case isUnprocessableModelError(err):
+	case IsUnprocessableModelError(err):
 		return NewUnprocessableEntityError().Wrap(err)
-	case isConflictError(err):
+	case IsConflictError(err):
 		return NewConflictError().Wrap(err)
-	case isBadRequestError(err):
+	case IsBadRequestError(err):
 		return NewBadRequestError().Wrap(err)
 	default:
 		return NewServerError().Wrap(err)
 	}
 }
 
-func isNotFoundError(err error) bool {
+func IsNotFoundError(err error) bool {
```

Comment:
I made those functions in /transport/errors/errrormapper public because there may be cases when I want to check the error type in the handler (as you can see in `webhook` handler)